EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,10 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+uploader.*##a[href*="https://shopcart4me.com"]
+uploadflix.*##a[href^="https://t.co/"]
+uploadraja.*##a[target="_blank"]
```

Comment:
Too wide rules. Why not `uploadflix.org,uploadraja.com##center > a[target="_blank"] > img`?